core/aws_ddk_core/resources/_s3.py

```diff
@@ -31,6 +31,7 @@ class BucketSchema(BaseSchema):
     versioned = fields.Bool(load_default=True)
     auto_delete_objects = fields.Bool()
     enforce_ssl = fields.Bool(load_default=True)
+    event_bridge_enabled = fields.Bool(load_default=True)
```

Comment:
Do you think users will actually want to enable this differently from one env to the other?

Reply:
Good point, unlikely -- I'll remove it